### PR TITLE
128T Release notes 4.4.2

### PR DESCRIPTION
**Pull request by MichaelBaj**

### CODE REVIEW

File: docs/release_notes_128t_4.4.md [hunk 1]
```diff
@@ -3,6 +3,145 @@ title: 128T 4.4 Release Notes
...
+## Release 4.4.2
+
+- **I95-18807** Innocuous error produced in journal due to imudp module loaded by rsyslog daemon
+
+  _**Symptoms:**_ The following message can be seen in the journal
+  ```
+  rsyslogd[1337]: imudp: module loaded, but no listeners defined - no input will be gathered [v8.24.0 try http://www.rsyslog.com/e/2212 ]
+  ```
+------
+- **I95-32594** Validation allows for mismatched adjacency security-policy with peer network-interface security-policy for cases where multiple network interfaces in a router have the same IP Address. Only the first one is considered for matching inter-router-security policy between the network interface and peer adjacency.
```
Comment (ptimmons) [inline docs/release_notes_128t_4.4.md:15]: IP address

File: docs/release_notes_128t_4.4.md [hunk 2]
```diff
@@ -3,6 +3,145 @@ title: 128T 4.4 Release Notes
...
+
+  _**Conditions:**_ When multiple NTP servers are configured, at least one is reachable and at least one is not reachable
+------
+- **I95-35193** Performing a download of software may fail
+
+  _**Conditions**_ 128T connection to the conductor is disconnected or restarted
+------
+- **I95-35331** A custom chart that contains multiple line charts selects the incorrect graph when clicking on the corresponding legend
+------
+- **I95-35544** LTE SIM number (ICCID) is absent from the output of `show device interface` on LTE interfaces
```
Comment (ptimmons) [inline docs/release_notes_128t_4.4.md:43]: The command is "show device-interface"

File: docs/release_notes_128t_4.4.md [hunk 3]
```diff
@@ -3,6 +3,145 @@ title: 128T 4.4 Release Notes
...
+  ```
+  rm -f /var/lib/rpm/__*
+  rpm --rebuilddb
+  ```
+------
+- **I95-35793** Large responses from a DNS server may be rejected by 128T. When this happens, provisioned FQDNs remain unresolved.
+
+  _**Conditions:**_ The following log message can be seen:
+  ```
+  Jun 16 06:09:25.272 [DNS |DNSR] WARN (dnsManagerTP ) Failed to parse Ipv4Host (1) response for edge-global.plcm.vc: Message too long
```
Comment (ptimmons) [inline docs/release_notes_128t_4.4.md:57]: Is this FQDN one that ambiFOX was trying to resolve? If so, we should replace it with something like "dummy-address.128technology.com"

File: docs/release_notes_128t_4.4.md [hunk 4]
```diff
@@ -3,6 +3,145 @@ title: 128T 4.4 Release Notes
...
+  network[4650]: [FAILED]
+  ```
+------
+- **I95-36283** The 128T router asset state is stuck on its current state
+
+  _**Conditions:**_ The following log message can be seen:
+  ```
+  TypeError: heap argument must be a list
+  ```
+  Until the system is upgraded to 4.4.2, this issue can be mitigated by restarting the salt-minion service by executing `systemctl restart salt-minion` on the Linux shell. If not manually restarted, the salt-minion watchdog will also restart the salt-minion after one hour.
```
Comment (ptimmons) [inline docs/release_notes_128t_4.4.md:90]: ... `systemctl restart salt-minion` in the Linux shell.

**Review by jeffmart — DISMISSED** [review]

**Review by jeffmart — APPROVED** [review]

**Pull request merged**